packages/trpc/src/procedures/recipes/updateRecipe.spec.ts

```diff
@@ -0,0 +1,134 @@
+import { trpcSetup, tearDown } from "../../testutils";
+import { prisma } from "@recipesage/prisma";
+import { User } from "@prisma/client";
+import type { CreateTRPCProxyClient } from "@trpc/client";
+import type { AppRouter } from "../../index";
+import { recipeFactory } from "../../factories/recipeFactory";
+
+describe("updateRecipe", () => {
+  let user: User;
+  let trpc: CreateTRPCProxyClient<AppRouter>;
+
+  beforeAll(async () => {
+    ({ user, trpc } = await trpcSetup());
+  });
+
+  afterAll(() => {
+    return tearDown(user.id);
+  });
+
+  describe("success", () => {
+    it("updates the recipe", async () => {
+      const recipe = await prisma.recipe.create({
+        data: {
+          ...recipeFactory(user.id),
+          title: "Mexican chicken",
+        },
+      });
+      const response = await trpc.recipes.updateRecipe.mutate({
+        ...recipeFactory(user.id),
+        title: "marmelad",
+        id: recipe.id,
+        labelIds: [],
+        imageIds: [],
+        folder: "main",
+      });
+      expect(response.title).toEqual("marmelad");
```

Comment:
Would be good to query the DB again and check that it was properly updated in the database